p8_integration_tests/quick_test/test_auto_pause_and_resume_tests/test_shorter_run/test_spike_source_array.py

```diff
@@ -55,5 +56,7 @@ def recording_1_element(self):
 
         sim.end()
 
+    pytest.mark.timeout(12000)
```

Comment:
Should this be a decorator (with `@`)?

Reply:
think when we tried that, it didn't work. i settled on this one because it did work. 

As @Christian-B points out. it may be worth seeing if it still passes without it. But It may be that this specific test actually needs more time than normal.